baker/siteRenderers.tsx

```diff
@@ -602,16 +607,18 @@ const getExplorerTitleByUrl = async (url: Url): Promise<string | undefined> => {
     return explorer.explorerTitle
 }
 
-const renderGrapherImageByChartSlug = async (
+/**
+ * Renders a chart thumbnail given a slug. The slug is considered "resolved",
+ * meaning it has gone through the internal URL resolver and is final from a
+ * redirects perspective.
+ *
+ * @param chartSlug
+ * @returns
```

Comment:
nit: param/returns are already expressed in TypeScript? I think probably VSCode automatically added it to "help you out"